CHANGELOG.md

```diff
@@ -1,5 +1,107 @@
 # Lotus changelog
 
+# UNRELEASED
+
+# v1.23.3 / 2023-07-27
+
+This feature release of Lotus includes numerous improvements and enhancements for node operators, ETH RPC-providers and storage providers.
+
+## Highlights
+
+- The Ethereum-like RPC exposed by Lotus is now compatible with EIP-1898: https://github.com/filecoin-project/lotus/pull/10815
+- The lotus-miner PieceReader now supports parallel reads: https://github.com/filecoin-project/lotus/pull/10913
+- Added new environment variable `LOTUS_EXEC_TRACE_CACHE_SIZE` to configure execution trace cache size ([filecoin-project/lotus#10585](https://github.com/filecoin-project/lotus/pull/10585))
+  - If unset, we default to caching 16 most recent execution traces. Storage Providers may want to set this to 0, while exchanges may want to crank it up. 
+
+## New features
+  - Implement a tooling for slasher  ([filecoin-project/lotus#10928](https://github.com/filecoin-project/lotus/pull/10928))
```

Comment:
I think this deserves a highlight(one of the big goal we are trying to hit) & link to the use educational context